### PR TITLE
Create migration vote_products, vouchers

### PR DESCRIPTION
**Pull request by at-phuctv**
Create migration vote_products, 
Create migration vouchers

### CODE REVIEW

File: database/migrations/2016_11_28_151248_create_vote_products_table.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('vote_products', function (Blueprint $table) {
+            $table->integer('rank');
+            $table->datetime('date_created');
```
Comment (dzungnguyenh) [inline database/migrations/2016_11_28_151248_create_vote_products_table.php]: [rank A] why we need data_created there?

File: database/migrations/2016_11_28_151248_create_vote_products_table.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            $table->integer('user_id')->unsigned();
+            $table->integer('product_id')->unsigned();
```
Comment (dzungnguyenh) [inline database/migrations/2016_11_28_151248_create_vote_products_table.php:19]: [rank A] please add created_at & updated_at for this table

File: database/migrations/2016_11_28_151416_create_vouchers_table.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('vouchers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name',100)->unique();
+            $table->integer('poin');
+            $table->string('value',100);
+            $table->integer('user_id')->unsigned();
```
Comment (dzungnguyenh) [inline database/migrations/2016_11_28_151416_create_vouchers_table.php:21]: [rank A] please add created_at & updated_at for this table

File: database/migrations/2016_11_28_151416_create_vouchers_table.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('vouchers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name',100)->unique();
```
Comment (dzungnguyenh) [inline database/migrations/2016_11_28_151416_create_vouchers_table.php]: [rank A] length of vouchers name should be 255 chars, I think

File: database/migrations/2016_11_28_151416_create_vouchers_table.php [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+            $table->integer('poin');
```
Comment (dzungnguyenh) [inline database/migrations/2016_11_28_151416_create_vouchers_table.php]: [rank C] what does 'poin' mean? Please named it clearly and check spelling carefully.

File: database/migrations/2016_11_28_151416_create_vouchers_table.php [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+            $table->string('value',100);
```
Comment (dzungnguyenh) [inline database/migrations/2016_11_28_151416_create_vouchers_table.php]: I have a question: please explain for me about value meaning, why do you using string there?

**Review by dzungnguyenh — CHANGES_REQUESTED** [review]
I finished review. Please check my comments.

File: database/migrations/2016_11_28_151416_create_vouchers_table.php (hunk 6 above)
Comment (at-phuctv) [reply database/migrations/2016_11_28_151416_create_vouchers_table.php]: This is value of voucher.I will change the value of type float.

**Pull request closed without merging**